content/en/tracing/setup_overview/compatibility_requirements/dotnet-core.md

```diff
@@ -15,68 +15,84 @@ further_reading:
       tag: 'GitHub'
       text: 'Examples of Custom Instrumentation'
 ---
+<div class="alert alert-warning">
+  <strong>Notes:</strong><br><ul><li>Datadog automatic instrumentation relies on the .NET CLR Profiling API. This API allows only one subscriber (for example, APM). To ensure maximum visibility, run only one APM solution in your application environment.</li><li> If you are using both automatic and custom instrumentation, it is important to keep the package versions (for example, MSI and NuGet) in sync.</li></ul>
+</div>
```

Comment:
+1, and since this is now brought to my attention, I'd suggest moving this from the compatibility page to the setup page. That would make the most sense to me

Reply:
Updated on both .NET Core & Framework and moved this out of the compatibility page to the setup page under the Installation and getting started section.